src/v/cluster/members_manager.cc

```diff
@@ -1529,15 +1529,25 @@ members_manager::handle_configuration_update_request(
           err.value(),
           req.node,
           all_brokers);
-        return ss::make_ready_future<ret_t>(errc::invalid_configuration_update);
+        co_return errc::invalid_configuration_update;
+    }
+
+    try {
+        co_await update_broker_client(
+          _self.id(),
+          _connection_cache,
+          req.node.id(),
+          req.node.rpc_address(),
+          _rpc_tls_config);
+    } catch (...) {
+        vlog(
+          clusterlog.warn,
+          "Unable to handle configuration update due to broker update error: "
+          "{}",
+          std::current_exception());
+        co_return configuration_update_reply{false};
     }
 
-    auto f = update_broker_client(
```

Comment:
Previously, `f` was never awaited upon. I'm not sure if that was intended to be backgrounded, or if we want to wait for it, I'm assuming we should wait for it, (hence coroutinizing this method) but if the intention is to background it, then I'll happily replace this with `ssx::backgrounded`